src/main/java/org/mskcc/cmo/messaging/FileUtil.java

```diff
@@ -4,4 +4,5 @@
 
 public interface FileUtil {
     void savePublishFailureMessage(String topic, String message) throws IOException;
+    void writeToFile(String filename, String header, String value) throws IOException;
```

Comment:
Can you provide an example call here.  I'm not understanding what the args mean.

Reply:
filename (renaming that to filepath). is the fiel to write to, header is the file header, value is the data to write. Maybe I'll rename to filedata?